service/policy/db/query.sql

```diff
@@ -691,6 +691,31 @@ LEFT JOIN subject_condition_set scs ON scs.id = sm.subject_condition_set_id
 WHERE sm.id = $1
 GROUP BY av.id, sm.id, scs.id;
 
+-- name: MatchSubjectMappings :many
+SELECT
+    sm.id,
+    sm.actions,
+    JSON_STRIP_NULLS(JSON_BUILD_OBJECT('labels', sm.metadata -> 'labels', 'created_at', sm.created_at, 'updated_at', sm.updated_at)) AS metadata,
```

Comment:
Not a blocker, but we can probably forgo metadata for SMs and SCSs in this service to service API as they were probably only there originally due to reusing Squirrel-managed queries.

Reply:
Would you mind please opening an issue if you don't want to capture within this current work?

Reply:
https://github.com/opentdf/platform/issues/1673